### ISSUE TITLE
Current populate script has problems when two people have the same name

### ISSUE THREAD

**Issue by bradbeattie** [bradbeattie/canadian-parlimentarty-data#20]
```
HARVIE, John
	 General Election 6 (1887-02-22), Ontario: TORONTO CENTRE, HARVIE, John
	 General Election 41 (2011-05-02), Manitoba: WINNIPEG NORTH, HARVIE, John
```

**Comment by bradbeattie** [comment]
Or John Harvie is a Highlander...

**Comment by bradbeattie** [comment]
Merged into #18
